src/components/Labels/IDOLabels/IDOLabels.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from 'react'
+import classNames from 'classnames'
+import useStyle from './style'
+
+export interface IIDOLabelsProps {
+  className?: string
+  style?: React.CSSProperties
+  title: ReactElement | string
+  icon: string
+  value: string
+}
+export const IDOLabels: React.FC<IIDOLabelsProps> = ({ className, title, icon, value }) => {
+  const classes = useStyle()
+
+  return (
+    <div className={classNames(classes.general, className)}>
+      <label className={classes.title}>{title}</label>
```

Comment:
for texts use "Typography element" from material-ui; there is wrong font in stories for that component